frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.styled.tsx

```diff
@@ -19,9 +19,13 @@ export const ChevronDown = styled(Icon)`
 `;
 
 export const TriggerButton = styled(Button)<ButtonProps>`
-  color: ${() => alpha(color("text-white"), 0.5)};
   border-radius: 0;
-  border-left: 2px solid ${() => alpha(color("border"), 0.1)};
+
+  /** we hide the trigger button by default, and show it on hover */
+  visibility: hidden;
+  color: color-mix(in srgb, var(--mb-color-text-white), transparent 40%);
+  border-left: 2px solid
+    color-mix(in srgb, var(--mb-color-border), transparent 80%);
```

Comment:
We could probably use `lighten` instead of `alpha` to better support dark-mode theming, but I'll keep using `alpha` for now to maintain parity with main app. I modified the percentage slightly (e.g. `90% to 80%`) to make it a bit more legible in dark mode.